virtcontainers/kata_agent.go

```diff
@@ -743,6 +744,18 @@ func (k *kataAgent) createContainer(sandbox *Sandbox, c *Container) (p *Process,
 		return nil, err
 	}
 
+	for _, m := range c.mounts {
+		if utils.IsEphemeralStroage(m.Source) {
+			filename := filepath.Join(ephemeralPath, filepath.Base(m.Source))
+			teststorage := &grpc.Storage{
```

Comment:
`teststorage` is not a good variable name :)

Reply:
:) sorry. Leftover of an aftermath. I will clean it and update the patch.